apps/meteor/client/views/room/contextualBar/RoomMembers/RoomMembers.tsx

```diff
@@ -118,11 +118,7 @@ const RoomMembers = ({
 				{!loading && members.length > 0 && (
 					<Box pi='x18' pb='x12'>
 						<Box is='span' color='hint' fontScale='p2'>
-							{t('Showing')}: {members.length}
-						</Box>
-
-						<Box is='span' color='hint' fontScale='p2' mis='x8'>
-							{t('Total')}: {total}
+							{t('Showing')} {members.length} {t('of')} {total}
```

Comment:
Please, create a single translation key for the whole text.

Ex: `{t('Showing__current__of__total', { current: members.length, total })}`

Reply:
This is needed because in certain languages the order of verbs/nouns/connectors can change, and the only way to provide a correct translation would be to make a string for the whole text.

Reply:
Fixed